generated/nirfmxbluetooth/nirfmxbluetooth.proto

```diff
@@ -1367,24 +1367,28 @@ message InitializeRequest {
   string session_name = 1;
   string resource_name = 2;
   string option_string = 3;
+  nidevice_grpc.SessionInitializationBehavior initialization_behavior = 4;
 }
 
 message InitializeResponse {
   int32 status = 1;
   nidevice_grpc.Session instrument = 2;
   int32 is_new_session = 3;
   string error_message = 4 [deprecated = true];
+  bool new_session_initialized = 5;
```

Comment:
TODO: Create a test for this, namely to make sure that there's scenarios where `is_new_session` and `new_session_initialized` differ and thus should both be represented in the response.

My thought is that they can be different because `is_new_session` is driver related while `new_session_initialized` is just about if the library's init method was called successfully and a new gRPC session was created (new mapping between named session and session handle): 

1. Initialize with `resource_name`/`option_string` and `sessionName = "SessionName1"`
   1a. `isNewSession` is True and `newSessionInitialized` is True
2. Init with same `resource_name`/`option_string` and different `sessionName = "SessionName2"`
   2a. `isNewSession` from driver is False while `newSessionInitialized` is True

Reply:
Added `Init_InitAgain_NewSessionInitializedValuesDiffer` to `nirfmxbluetooth_driver_api_tests.cpp` that confirms this behavior and the need for both output fields.